assets/js/theme/search.js

```diff
@@ -132,8 +132,43 @@ export default class Search extends CatalogPage {
         $($tabsCollection.get(nextTabIdx)).focus().trigger('click');
     }
 
+    getUrlParameter(queryParam) {
+        const queryParamKey = queryParam.replace(/[\[]/, '\\[').replace(/[\]]/, '\\]');
```

Comment:
Why do we need to do this if you are passing 'search_query' string?

Reply:
This non absolutely necessary in this case. It was made for sterling realisation of such type of function

Reply:
For now I believe we should not add it, in case of reusing this function, is should be moved to utils and updated